docs/articles/documentation/guides/basic-guides/select-page-elements.md

```diff
@@ -0,0 +1,542 @@
+---
+layout: docs
+title: Select Page Elements
+permalink: /documentation/guides/basic-guides/select-page-elements.html
+---
+# Select Page Elements
+
+TestCafe requires that you identify the target page element to perform an [action](interact-with-the-page.md) with it (click, drag, etc.) or check its state in an [assertion](assert.md).
+
+You can pass a CSS selector to an action function to specify the target element:
+
+```js
+await t.click('#my-button');
+```
+
+For more complex queries, however, CSS selectors get longer and difficult to read, write, and maintain:
```

Comment:
Regarding 'complex' and 'difficult' - I don't think it's excessive. The idea is that *complex* queries can be accomplished with cumbersome CSS selectors (which are *difficult* to write) or simple TestCafe selectors (which is their main advantage).

Regarding matters of opinion - I'm aiming to explain why you should use TestCafe selectors, not just how. I know we're not doing this in DevExpress docs, and I understand that we can't write docs like Cypress (I wish we could), but I just want to write at least a couple of sentences saying why.

Reply:
This is what I say below. So, the logic is: (1) you can use CSS selectors, but (2) this is not always a good idea because it can be difficult so (3) TestCafe introduces its own selectors with these and that advantages. It seems rather straightforward to me.